tests/conftest.py

```diff
@@ -132,7 +131,10 @@ def fixture_case_id() -> str:
     return CaseIDs.RUNNING
 
 
-@pytest.fixture(name="tower_task")
+@pytest.fixture(name="tower_task", scope="session")
```

Comment:
👍 This fixture is probably pretty expensive to setup since it reads a file

Reply:
Yeah, I tried but pytest complains. Do not want to spend time on it now as we will remake the sql session handling in the future and the test suite is rather fast as it is.